app/models/user.rb

```diff
@@ -180,4 +183,10 @@ def role_display
   def require_volunteer_profile_attributes?
     (volunteer? || coordinator?) && invitation_accepted_at?
   end
+
+  def check_phone_verification
+    if phone_changed? && phone_was.present?
+      self.verified = false
+    end
```

Comment:
How do you feel about using a guard statement here?

```ruby
return unless phone_changed? && phone_was.present?

self.verified = false
```

Reply:
I feel like it's best to show it's only unverifying the user if their phone changes, which is a less common scenario than it not changing.